civihr_default_theme/scss/components/_leave-request-modal.scss

```diff
@@ -16,6 +16,70 @@ $row-margin-bottom: 15px;
   margin-bottom: 20px;
 }
 
+.chr_leave-request-modal__form {
+
+  .ui-select-container {
+    .text-muted {
+      color: $brand-primary;
+      font-size: $crm-font-size-large;
+    }
+
+    &:not(.ui-select-multiple) {
+      .glyphicon-remove {
+        top: 5px;
+      }
+
+      .ui-select-match-text span {
+        font-size: $crm-font-size-large;
+      }
+    }
+
+    input[type="search"] {
+      color: $brand-primary;
+      font-size: $crm-font-size-large;
+      border: none !important;
+    }
+  }
+
+  .form-control::-webkit-input-placeholder {
```

Comment:
Create a mixin on sites/all/themes/civihr_employee_portal_theme/civihr_default_theme/scss/base/_mixins.scss to remove all this browser specific code
```
@mixin optional-at-root($sel) {
  @at-root #{if(not &, $sel, selector-append(&, $sel))} {
    @content;
  }
}

@mixin placeholder {
  @include optional-at-root('::-webkit-input-placeholder') {
    @content;
  }

  @include optional-at-root(':-moz-placeholder') {
    @content;
  }

  @include optional-at-root('::-moz-placeholder') {
    @content;
  }

  @include optional-at-root(':-ms-input-placeholder') {
    @content;
  }
}
```
use it here
```
.chr_ui-select--info-colored {
  @include placeholder {
    color: $brand-primary;
  }
}